dropwizard-health/src/main/java/io/dropwizard/health/response/ServletHealthResponderFactory.java

```diff
@@ -0,0 +1,53 @@
+package io.dropwizard.health.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.health.HealthEnvironment;
+import io.dropwizard.jersey.setup.JerseyEnvironment;
+import io.dropwizard.jetty.setup.ServletEnvironment;
+
+import java.util.Collection;
+
+/**
+ * A servlet-based implementation of {@link HealthResponderFactory}, to respond to health check requests.
+ */
+@JsonTypeName("servlet")
+public class ServletHealthResponderFactory implements HealthResponderFactory {
+    static final String SERVLET_SUFFIX = "-servlet";
+
+    @JsonProperty
+    private boolean cacheControlEnabled = true;
+
+    @JsonProperty
+    private String cacheControlValue = "no-store";
+
+    public boolean isCacheControlEnabled() {
+        return cacheControlEnabled;
+    }
+
+    public void setCacheControlEnabled(boolean cacheControlEnabled) {
+        this.cacheControlEnabled = cacheControlEnabled;
+    }
+
+    public String getCacheControlValue() {
+        return cacheControlValue;
+    }
+
+    public void setCacheControlValue(String cacheControlValue) {
+        this.cacheControlValue = cacheControlValue;
+    }
+
+    @Override
+    public void configure(final String name, final Collection<String> healthCheckUrlPaths,
+                          final HealthResponseProvider healthResponseProvider,
+                          final HealthEnvironment health, final JerseyEnvironment jersey,
+                          final ServletEnvironment servlets, final ObjectMapper mapper) {
+        final ServletHealthResponder servlet = new ServletHealthResponder(healthResponseProvider, cacheControlEnabled,
+            cacheControlValue);
+        servlets
+            .addServlet(name + SERVLET_SUFFIX, servlet)
+            .addMapping(healthCheckUrlPaths.toArray(new String[0]));
+        health.setHealthResponder(servlet);
```

Comment:
My reasoning here is because I was trying to rationalize how this would all work if I wanted to plug in something like a `GrpcResponder`. Since gRPC isn't at all plugged into the main Dropwizard framework, I wanted a way to access whatever responder we produced from the `HealthEnvironment` so I could plug it where necessary

Reply:
Decided to remove this for now. If there's some value in adding it in the future, I'll circle back